_spinnaker_install_admin_guides/install-on-k8s.md

```diff
@@ -0,0 +1,710 @@
+---
+layout: post
+title: Installing Spinnaker in Kubernetes
+order: 21
+published: true
+redirect_from:
+  - /spinnaker_install_admin_guides/install_on_k8s/
+  - /spinnaker_install_admin_guides/install-on-k8s/
+  - /spinnaker-install-admin-guides/install_on_k8s/
+---
+
+This guide describes how to install Spinnaker in Kubernetes.  It assumes that you have the following:
+
```

Comment:
Is this assumption list necessary here? There's a whole heading a little further down aout it.